src/appleseed.shaders/include/appleseed/math/as_math_helpers.h

```diff
@@ -77,52 +75,17 @@ float fast_gain(float value, float g)
         : fast_bias(value * 2 - 1, 1 - g) * 0.5 + 0.5;
 }
 
-
 //
 // Reference:
 //
-//      Lehmer linear congruential generator (Park-Miller)
```

Comment:
Out of curiosity, why did you remove the RNG stuff?

Reply:
It was only used previously in the implementation of the maya specific procedural noises, and we replaced that by OSL's native hashnoise(), which allowed us also to discard the big arrays/tables. If it's handy to have the RNG in general (albeit unused atm) let me know, i'll reinstate it.

Reply:
It was only used previously in the implementation of the maya specific procedural noises. There it was replaced by OSL's native **hashnoise()**, which allowed us also to discard the big arrays/tables. If it's handy to have the (pseudo) RNG in general (albeit unused atm) let me know so i can reinstate it.

Reply:
Was just curious, thanks for the explanation!